testnet/tools/nns-tools/list-new-commits.sh

```diff
@@ -19,39 +19,86 @@ Usage: $0 (<COMMIT_ID>)
     exit 1
 }
 
-if [ $# -gt 1 ]; then
+RELEASE_CANDIDATE_COMMIT_ID="${1:-GARBAGE}"
+
+if [[ "${RELEASE_CANDIDATE_COMMIT_ID}" == 'GARBAGE' ]]; then
```

Comment:
Why not just use the `latest_commit_with_prebuilt_artifacts`?  

Now I have to run the command, copy/paste output, and run it again - seems unnecessary.